File: Assignment Programs/Assignment46/program4.c
#include<stdio.h>

int Min(int iNo)
{
	static int iMin = 10;
	int iDigit = 0;
	
	if(iNo < 0)
	{
		iNo = -iNo;
	}

	if(iNo != 0)
	{
		iDigit = iNo % 10;
		if(iDigit < iMin)
		{
			iMin = iDigit;
		}
		iNo = iNo / 10;
		Min(iNo);
	}
	return iMin;
}

int main()
{
	int iRet = 0, iValue = 0;
	printf("Enter number\n");
	scanf("%d",&iValue);
	iRet = Min(iValue);
	printf("Minimum digit from the number is : %d\n",iRet);
	return 0;
}

File: Assignment Programs/Assignment33/program4.java
import java.util.*;

class Digit
{
	public int Multiply(int iNo)
	{
		int iDigit = 0, iMult = 1;
		
		while(iNo != 0)
		{
			iDigit = iNo % 10;
			iMult = iDigit * iMult;
			iNo = iNo / 10;
		}
		return iMult;
	}
}

class program4
{
	public static void main(String args[])
	{
		int iValue = 0, iRet = 0;
		Scanner sobj = new Scanner(System.in);
		System.out.println("Enter the number");
		iValue = sobj.nextInt();
	
		Digit dobj = new Digit();
		
		iRet = dobj.Multiply(iValue);
	
		System.out.println("Multiplication of all digits is : "+iRet);
	}
}